com.unity.render-pipelines.universal/Shaders/2D/Include/SpriteMaskShared.hlsl

```diff
@@ -0,0 +1,45 @@
+// alpha below which a mask should discard a pixel, thereby preventing the stencil buffer from being marked with the Mask's presence
+half  _Cutoff;
+half4 _Color;
+half4 _RendererColor;
+half2 _Flip;
+
+TEXTURE2D(_MainTex);
+SAMPLER(sampler_MainTex);
+
+struct appdata_masking
```

Comment:
As this is a URP shader, we should use the new convention and call this Attributes.

Reply:
Fixed